.github/workflows/publish_tag.yml

```diff
@@ -114,4 +114,4 @@ jobs:
           # For posting a simple plain text message
           slack-message: "Schematic has just been released. Check out new version: ${{ github.ref_name }}"
         env:
-          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
```

Comment:
Also, if this workflow is for triggering a dev release, I don't think we need to post a message on slack

Reply:
Removed that